backend/src/database/seeds/01_users.js

```diff
@@ -2,30 +2,36 @@
 exports.seed = function(knex) {
```

Comment:
Is there a reason the migrations and seeds aren't using arrow functions? I think we should be striving for syntactic consistency, not to mention the two function notations work slightly differently. Arrow functions would be handy here to lesson the code footprint of these files.

Reply:
this is a good point. the only reason i didn't use them is because the seed generator makes them with the old syntax by default, so i just kept it because it is only a seeder. arrow functions would clean this right up though, so i'm implementing those. good catch!